float32 fixed_mast_yaw
---
bool success
string message